AndelaOSP/andela-societies-backend
# #161471006 Improve code maintainability

## Resolves #161471006

## Description (what problem you're fixing)

  - The current code maintainability is poor and there exists test files with large codebases that are not closely related

## Fix (what you did to fix it)

  - Broke down large test files into smaller related modules. 
  - Updated Codeclimate `similar-code mass threshold` to 50

## How to test (describe how to test your PR)

- Clone the repository, install docker, run `make test` within the repository.
- Trigger a build on CircleCI.
- An improvement on code maintainability should be noted on code-climate.
